nmi_mysql/nmi_mysql.py

```diff
@@ -134,6 +149,8 @@ def _execute(self, _query, _params, _multi=False):
                         'affected_rows': result.rowcount
                     }
 
+            con.close()
```

Comment:
Ah, I see now.. Hum, I guess we don't give out connection at all, which means we also cannot do transactions? I guess this is fine for now since we don't use transactions?